Persephone/Controllers/AlbumViewController.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  AlbumViewController.swift
+//  Persephone
+//
+//  Created by Daniel Barber on 2019/2/08.
+//  Copyright © 2019 Dan Barber. All rights reserved.
+//
+
+import Cocoa
+
+class AlbumViewController: NSViewController,
+                           NSCollectionViewDataSource,
+                           NSCollectionViewDelegate,
+                           NSCollectionViewDelegateFlowLayout {
+  var albums: [MPDClient.Album] = []
+  var albumWidth: CGFloat = 0
```

Comment:
Does `albumWidth` change, or should it be a `let`?
Also, is it used externally? (I didn't clone the repo so I'm just looking at these commits).

Reply:
Actually I think this was leftover from a refactor I tried and isn't required any more.